_data/providers.json

```diff
@@ -29,7 +29,7 @@
     {
       "name": "Cubes Hosting",
       "url": "https://www.cubes.host/",
-      "description": "Install Geyser using the plugin manager. Then restart the server and Geyser will run on an additional port - you can check it in the server console. Geyser-Standalone can be set up through a support ticket."
+      "description": "Order a Minecraft Cross-Play (GeyserMC) plan and directly have access to a paper server with GeyserMC pre-installed. Configure the second port in the GeyserMC config that came with the service and restart the service. Now you can connect through both the Java and Bedrock Editions of Minecraft. Geyser-Standalone can be set up through a support ticket"
```

Comment:
Sorry, but that is really not helpful.. what is the `second port` here? Is that the `bedrock port`, or the `java port` in the config? If it's the latter, Geyser auto-detects that so that does not need to be set.
If your host is doing an automatic configuration - e.g. using [Geyser's startup args](https://wiki.geysermc.org/geyser/geyser-command-line-arguments-and-system-properties/) - users shouldnt need to change the ip/port. I'd just like clarification whether Bedrock players would connect with the same port as Java players, or a separate one :)